app/src/main/kotlin/com/simplemobiletools/draw/pro/helpers/Constants.kt

```diff
@@ -5,6 +5,7 @@ const val CANVAS_BACKGROUND_COLOR = "canvas_background_color"
 const val SHOW_BRUSH_SIZE = "show_brush_size"
 const val BRUSH_SIZE = "brush_size_2"
 const val LAST_SAVE_FOLDER = "last_save_folder"
+const val LAST_SAVE_EXT = "last_save_ext"
```

Comment:
change "ext" to "extension" please, avoid such abbreviations

Reply:
I did a fix in 29128cd